---
layout: cv
title: Egon Willighagen's CV
---

# Egon Willighagen
Assistant professor, Maastricht University.

<i>2025-04-08</i>

<a href="egon.willighagen@maastrichtuniversity.nl">egon.willighagen@maastrichtuniversity.nl</a>

<div id="webaddress">
  <a href="https://orcid.org/0000-0001-7542-0286"><i class="fa-brands fa-orcid"></i> 0000-0001-7542-0286</a> - 
  <a href="https://chem-bla-ics.linkedchemistry.info/"><i class="fa-solid fa-blog"></i> chem-bla-ics.linkedchemistry.info</a> - 
  <!-- a href="https://egonw.github.io"><i class="fa-solid fa-house"></i> egonw.github.io</a> - --> <br />
  <a href="https://github.com/egonw"><i class="fa-brands fa-github"></i> egonw</a> - 
  <a href="https://www.linkedin.com/in/egon-willighagen/"><i class="fa-brands fa-linkedin"></i> egon-willighagen</a> - 
  <a href="https://social.edu.nl/@egonw"><i class="fa-brands fa-mastodon"></i> egonw</a> - 
  <a href="https://bsky.app/profile/egonw.social.edu.nl.ap.brid.gy"><i class="fa-brands fa-bluesky"></i> @egonw.ap.brid.gy</a>
</div>

## Education

`2002-2008`
**Ph.D., Chemistry**, Representation of Molecules and Molecular Systems in Data Analysis and Modeling, *Radboud University*, Nijmegen, The Netherlands. <i class="fa-solid fa-building-columns"></i> [isbn:9789090228068](https://hdl.handle.net/2066/72267)

`1993-2001`
**M.Sc., Chemistry**, *University of Nijmegen*, Nijmegen, The Netherlands

## Experience

`2024-present`
**Assistant Professor**, *Department of Translational Genomics, Institute NUTRIM for Nutrition and Translational Research in Metabolism, Maastricht University*, Maastricht, The Netherlands.

`2014-2024`
**Assistant Professor**, *Department of Bioinformatics, Maastricht University*, Maastricht, The Netherlands.

`2012-2014`
**Postdoc**, *Department of Bioinformatics, Maastricht University*, Maastricht, The Netherlands.

`2011`
**Postdoc**, *Institute for Environmental Medicine, Karolinska Institutet*, Stockholm, Sweden.

`2010`
**Postdoc**, *Unilever Center for Molecular Molecular Informatics, University of Cambridge*, Cambridge, United Kingdom.

`2008-2010`
**Postdoc**, *Department of Pharmaceutical Biosciences, Uppsala University*, Uppsala, Sweden.

`2007-2008`
**Postdoc**, *Plant Research International, Wageningen University
& Research Centre*, Wageningen, The Netherlands.

`2006-2007`
**Postdoc**, *Cologne University Bioinformatics Centre, Cologne
University*, Cologne, Germany.


## Publications

**Google Scholar** [u8SjMZ0AAAAJ](https://scholar.google.fi/citations?user=u8SjMZ0AAAAJ). H-index: 51.

**Web of Science** H-index: 41. Citations: 7,248 (without self: 6,896). Highly Cited Papers: 8. Citations per article: 51.04

**NIH iCite**: Mean RCR: 3.16. Articles in NIH Top 1%: 5, Top 10%: 16

### Journal articles

Scientific publications in international journals with a referee system.

`2025`
Kienhuis, A., Krul, C., van Engelen, J., Evelo, C., Hessel, E., Hoekman, J., Kramer, N., Krop, E., Masereeuw, R., Moors, E., Negro, S., Piersma, A., Pieters, R., Teunis, M., **Willighagen, E.**, Legler, J. (2025). The Virtual Human Platform for Safety Assessment (VHP4Safety) project: Next generation chemical safety assessment based on human data. (2024). *ALTEX*. doi:[10.14573/altex.2407211](https://doi.org/10.14573/altex.2407211)

`2024`
Agrawal, A., Balcı, H., Hanspers, K., Coort, S. L., Martens, M., Slenter, D. N., Ehrhart, F., Digles, D., Waagmeester, A., Wassink, I., Abbassi-Daloii, T., Lopes, E. N., Iyer, A., Acosta, J. M., Willighagen, L. G., Nishida, K., Riutta, A., Basaric, H., Evelo, C. T., … Pico, A. R. (2023). WikiPathways 2024: next generation pathway database. *Nucleic Acids Research*, *52*(D1), D679–D689. doi:[10.1093/nar/gkad960](https://doi.org/10.1093/nar/gkad960)

Serrano, B. A., Gheorghe, L. C., Exner, T. E., Resch, S., Wolf, C., Himly, M., Falk, A., Bossa, N., Vazquez, S., Papadiamantis, A. G., Afantitis, A., Melagraki, G., Maier, D., Saramveis, H., **Willighagen, E.**, Lobaskin, V., Oldfield, J. D., & Lynch, I. (2024). The role of FAIR nanosafety data and nanoinformatics in achieving the UN sustainable development goals: the NanoCommons experience. *RSC Sustainability*, *2*(5), 1378–1399. doi:[10.1039/d3su00148b](https://doi.org/10.1039/d3su00148b)

Jeliazkova, N., Longhin, E., El Yamani, N., Rundén-Pran, E., Moschini, E., Serchi, T., Vrček, I. V., Burgum, M. J., Doak, S. H., Cimpan, M. R., Rios-Mondragon, I., Cimpan, E., Battistelli, C. L., Bossa, C., Tsekovska, R., Drobne, D., Novak, S., Repar, N., Ammar, A., … Dusinska, M. (2024). A template wizard for the cocreation of machine-readable data-reporting to harmonize the evaluation of (nano)materials. *Nature Protocols*, *19*(9), 2642–2684. doi:[10.1038/s41596-024-00993-1](https://doi.org/10.1038/s41596-024-00993-1)

Niarakis, A., Ostaszewski, M., Mazein, A., Kuperstein, I., Kutmon, M., Gillespie, M. E., Funahashi, A., Acencio, M. L., Hemedan, A., Aichem, M., Klein, K., Czauderna, T., Burtscher, F., Yamada, T. G., Hiki, Y., Hiroi, N. F., Hu, F., Pham, N., Ehrhart, F., … Schneider, R. (2024). Drug-target identification in COVID-19 disease mechanisms using computational systems biology approaches. *Frontiers in Immunology*, *14*. doi:[10.3389/fimmu.2023.1282859](https://doi.org/10.3389/fimmu.2023.1282859)

Ammar, A., Evelo, C., & **Willighagen, E.** (2024). FAIR assessment of nanosafety data reusability with community standards. *Scientific Data*, *11*(1). doi:[10.1038/s41597-024-03324-x](https://doi.org/10.1038/s41597-024-03324-x)

Slenter, D., Kutmon, M., Evelo, C. T., & **Willighagen, E. L.** (2024). Discovering life’s directed metabolic (sub)paths to interpret human biochemical markers using the DSMN tool. *Digital Discovery*, *3*(1), 81–98. doi:[10.1039/d3dd00069a](https://doi.org/10.1039/d3dd00069a)

Van Rijn, J. P. M., Martens, M., Ammar, A., Cimpan, M. R., Fessard, V., Hoet, P., Jeliazkova, N., Murugadoss, S., Vinković Vrček, I., & **Willighagen, E. L.** (2024). From papers to RDF-based integration of physicochemical data and adverse outcome pathways for nanomaterials. *Journal of Cheminformatics*, *16*(1). doi:[10.1186/s13321-024-00833-0](https://doi.org/10.1186/s13321-024-00833-0)

Bernabé, C. H., Thielemans, L., Kaliyaperumal, R., Carta, C., Zhang, S., van Gelder, C. W. G., Benis, N., da Silva Santos, L. O. B., Cornet, R., dos Santos Vieira, B., Lalout, N., Henriques, I., Ballesteros, A. C., Burger, K., Kersloot, M. G., Ehrhart, F., van Enckevort, E., Evelo, C. T., Gray, A. J. G., …, **Willighagen E. L.**, Jansen, M., Mons, B., Roos, M., Jacobsen, A. (2024). Building Expertise on FAIR Through Evolving Bring Your Own Data (BYOD) Workshops: Describing the Data, Software, and Management-focused Approaches and Their Evolution. *Data Intelligence*, *6*(2), 429–456. doi:[10.1162/dint_a_00236](https://doi.org/10.1162/dint_a_00236)

Murugadoss, S., Vinković Vrček, I., Schaffert, A., Paparella, M., Pem, B., Sosnowska, A.,  Stępnik, M., Martens, M., **Willighagen, E. L.**, Puzyn, T., Roxana Cimpan, M., Lemaire, F., Mertens, B., Dusinska, M., Fessard, V., Hoet, P.H.. Linking nanomaterial-induced mitochondrial dysfunction to existing adverse outcome pathways for chemicals. (2024). *ALTEX*, *41*(1). doi:[10.14573/altex.2305011](https://doi.org/10.14573/altex.2305011)

Groenewold, M., Bleeker, E. A. J., Noorlander, C. W., Sips, A. J. A. M., van der Zee, M., Aitken, R. J., Baker, J. H., Bakker, M. I., Bouman, E. A., Doak, S. H., Drobne, D., Dumit, V. I., Florin, M.-V., Fransman, W., Gonzalez, M. M., Heunisch, E., Isigonis, P., Jeliazkova, N., Jensen, K. A., … Scott-Fordsmand, J. J. (2024). Governance of advanced materials: Shaping a safe and sustainable future. *NanoImpact*, *35*, 100513. doi:[10.1016/j.impact.2024.100513](https://doi.org/10.1016/j.impact.2024.100513)

`2023`
Martens, M., Stierum, R., Schymanski, E. L., Evelo, C. T., Aalizadeh, R., Aladjov, H., Arturi, K., Audouze, K., Babica, P., Berka, K., Bessems, J., Blaha, L., Bolton, E. E., Cases, M., Damalas, D. Ε., Dave, K., Dilger, M., Exner, T., Geerke, D. P., … **Willighagen, E. L.** (2023). ELIXIR and Toxicology: a community in development. *F1000Research*, *10*, 1129. doi:[10.12688/f1000research.74502.2](https://doi.org/10.12688/f1000research.74502.2)

Mullowney, M. W., Duncan, K. R., Elsayed, S. S., Garg, N., van der Hooft, J. J. J., Martin, N. I., Meijer, D., Terlouw, B. R., Biermann, F., Blin, K., Durairaj, J., Gorostiola González, M., Helfrich, E. J. N., Huber, F., Leopold-Messer, S., Rajan, K., de Rond, T., van Santen, J. A., Sorokina, M., … Medema, M. H. (2023). Artificial intelligence for natural product drug discovery. *Nature Reviews Drug Discovery*, *22*(11), 895–916. doi:[10.1038/s41573-023-00774-7](https://doi.org/10.1038/s41573-023-00774-7)

Furxhi, I., **Willighagen, E.**, Evelo, C., Costa, A., Gardini, D., & Ammar, A. (2023). A data reusability assessment in the nanosafety domain based on the NSDRA framework followed by an exploratory quantitative structure activity relationships (QSAR) modeling targeting cellular viability. *NanoImpact*, *31*, 100475. doi:[10.1016/j.impact.2023.100475](https://doi.org/10.1016/j.impact.2023.100475)

Ammar, A., Cavill, R., Evelo, C., & **Willighagen, E.** (2023). PSnpBind-ML: predicting the effect of binding site mutations on protein-ligand binding affinity. *Journal of Cheminformatics*, *15*(1). doi:[10.1186/s13321-023-00701-3](https://doi.org/10.1186/s13321-023-00701-3)

**Willighagen, E.** (2023). Two years of explicit CiTO annotations. *Journal of Cheminformatics*, *15*(1). doi:[10.1186/s13321-023-00683-2](https://doi.org/10.1186/s13321-023-00683-2)

Rocca-Serra, P., Gu, W., Ioannidis, V., Abbassi-Daloii, T., Capella-Gutierrez, S., Chandramouliswaran, I., Splendiani, A., Burdett, T., Giessmann, R. T., Henderson, D., Batista, D., Emam, I., Gadiya, Y., Giovanni, L., **Willighagen, E.**, Evelo, C., Gray, A. J. G., Gribbon, P., Juty, N., … Sansone, S.-A. (2023). The FAIR Cookbook - the essential resource for and by FAIR doers. *Scientific Data*, *10*(1). doi:[10.1038/s41597-023-02166-3](https://doi.org/10.1038/s41597-023-02166-3)

Slenter, D. N., Hemel, I. M. G. M., Evelo, C. T., Bierau, J., **Willighagen, E. L.**, & Steinbusch, L. K. M. (2023). Extending inherited metabolic disorder diagnostics with biomarker interaction visualizations. *Orphanet Journal of Rare Diseases*, *18*(1). doi:[10.1186/s13023-023-02683-9](https://doi.org/10.1186/s13023-023-02683-9)

`2022`
Ammar, A., Cavill, R., Evelo, C., & **Willighagen, E.** (2022). PSnpBind: a database of mutated binding site protein–ligand complexes constructed using a multithreaded virtual screening workflow. *Journal of Cheminformatics*, *14*(1). doi:[10.1186/s13321-021-00573-5](https://doi.org/10.1186/s13321-021-00573-5)

Hoyt, C. T., Balk, M., Callahan, T. J., Domingo-Fernández, D., Haendel, M. A., Hegde, H. B., Himmelstein, D. S., Karis, K., Kunze, J., Lubiana, T., Matentzoglu, N., McMurry, J., Moxon, S., Mungall, C. J., Rutz, A., Unni, D. R., **Willighagen, E.**, Winston, D., & Gyori, B. M. (2022). Unifying the identification of biomedical entities with the Bioregistry. *Scientific Data*, *9*(1). doi:[10.1038/s41597-022-01807-3](https://doi.org/10.1038/s41597-022-01807-3)

Mohammed Taha, H., Aalizadeh, R., Alygizakis, N., Antignac, J.-P., Arp, H. P. H., Bade, R., Baker, N., Belova, L., Bijlsma, L., Bolton, E. E., Brack, W., Celma, A., Chen, W.-L., Cheng, T., Chirsir, P., Čirka, Ľ., D’Agostino, L. A., Djoumbou Feunang, Y., Dulio, V., … Schymanski, E. L. (2022). The NORMAN Suspect List Exchange (NORMAN-SLE): facilitating European and worldwide collaboration on suspect screening in high resolution mass spectrometry. *Environmental Sciences Europe*, *34*(1). doi:[10.1186/s12302-022-00680-6](https://doi.org/10.1186/s12302-022-00680-6)

Sinclair, G., Thillainadarajah, I., Meyer, B., Samano, V., Sivasupramaniam, S., Adams, L., **Willighagen, E. L.**, Richard, A. M., Walker, M., & Williams, A. J. (2022). Wikipedia on the CompTox Chemicals Dashboard: Connecting Resources to Enrich Public Chemical Data. *Journal of Chemical Information and Modeling*, *62*(20), 4888–4905. doi:[10.1021/acs.jcim.2c00886](https://doi.org/10.1021/acs.jcim.2c00886)

Jacobs, A., Williams, D., Hickey, K., Patrick, N., Williams, A. J., Chalk, S., McEwen, L., **Willighagen, E.**, Walker, M., Bolton, E., Sinclair, G., & Sanford, A. (2022). CAS Common Chemistry in 2021: Expanding Access to Trusted Chemical Information for the Scientific Community. *Journal of Chemical Information and Modeling*, *62*(11), 2737–2743. doi:[10.1021/acs.jcim.2c00268](https://doi.org/10.1021/acs.jcim.2c00268)

Krans, N. A., Ammar, A., Nymark, P., **Willighagen, E. L.**, Bakker, M. I., & Quik, J. T. K. (2022). FAIR assessment tools: evaluating use and performance. *NanoImpact*, *27*, 100402. doi:[10.1016/j.impact.2022.100402](https://doi.org/10.1016/j.impact.2022.100402)

Miller, R. A., Kutmon, M., Bohler, A., Waagmeester, A., Evelo, C. T., & **Willighagen, E. L.** (2022). Understanding signaling and metabolic paths using semantified and harmonized information about biological interactions. *PLOS ONE*, *17*(4), e0263057. doi:[10.1371/journal.pone.0263057](https://doi.org/10.1371/journal.pone.0263057)

Rutz, A., Sorokina, M., Galgonek, J., Mietchen, D., **Willighagen, E.**, Gaudry, A., Graham, J. G., Stephan, R., Page, R., Vondrášek, J., Steinbeck, C., Pauli, G. F., Wolfender, J.-L., Bisson, J., & Allard, P.-M. (2022). The LOTUS initiative for open knowledge management in natural products research. *eLife*, *11*. doi:[10.7554/elife.70780](https://doi.org/10.7554/elife.70780)

Van Rijn, J., Afantitis, A., Culha, M., Dusinska, M., Exner, T. E., Jeliazkova, N., Longhin, E. M., Lynch, I., Melagraki, G., Nymark, P., Papadiamantis, A. G., Winkler, D. A., Yilmaz, H., & **Willighagen, E.** (2022). European Registry of Materials: global, unique identifiers for (undisclosed) nanomaterials. *Journal of Cheminformatics*, *14*(1). doi:[10.1186/s13321-022-00614-7](https://doi.org/10.1186/s13321-022-00614-7)

Martens, M., Evelo, C. T., & **Willighagen, E. L.** (2022). Providing Adverse Outcome Pathways from the AOP-Wiki in a Semantic Web Format to Increase Usability and Accessibility of the Content. *Applied In Vitro Toxicology*, *8*(1), 2–13. doi:[10.1089/aivt.2021.0010](https://doi.org/10.1089/aivt.2021.0010)

Mortensen, H. M., Martens, M., Senn, J., Levey, T., Evelo, C. T., **Willighagen, E. L.**, & Exner, T. (2022). The AOP-DB RDF: Applying FAIR Principles to the Semantic Integration of AOP Data Using the Research Description Framework. *Frontiers in Toxicology*, *4*. doi:[10.3389/ftox.2022.803983](https://doi.org/10.3389/ftox.2022.803983)

Ni, Z., Wölk, M., Jukes, G., Mendivelso Espinosa, K., Ahrends, R., Aimo, L., Alvarez-Jarreta, J., Andrews, S., Andrews, R., Bridge, A., Clair, G. C., Conroy, M. J., Fahy, E., Gaud, C., Goracci, L., Hartler, J., Hoffmann, N., Kopczyinki, D., Korf, A., … Fedorova, M. (2022). Guiding the choice of informatics software and tools for lipidomics research applications. *Nature Methods*, *20*(2), 193–204. doi:[10.1038/s41592-022-01710-0](https://doi.org/10.1038/s41592-022-01710-0)

Sochorová, M., Vávrová, K., Fedorova, M., Ni, Z., Slenter, D., Kutmon, M., **Willighagen, E. L.**, Letsiou, S., Töröcsik, D., Marchetti-Deschmann, M., Zoratto, S., Kremslehner, C., & Gruber, F. (2022). Research Techniques Made Simple: Lipidomic Analysis in Skin Research. *Journal of Investigative Dermatology*, *142*(1), 4-11.e1. doi:[10.1016/j.jid.2021.09.017](https://doi.org/10.1016/j.jid.2021.09.017)

`2021`
Ostaszewski, M., Niarakis, A., Mazein, A., Kuperstein, I., Phair, R., Orta‐Resendiz, A., Singh, V., Aghamiri, S. S., Acencio, M. L., Glaab, E., Ruepp, A., Fobo, G., Montrone, C., Brauner, B., Frishman, G., Monraz Gómez, L. C., Somers, J., Hoch, M., Kumar Gupta, S., … Schneider, R. (2021). COVID‐19 Disease Map, a computational knowledge repository of virus‐host interaction mechanisms. *Molecular Systems Biology*, *17*(12). doi:[10.15252/msb.202110851](https://doi.org/10.15252/msb.202110851)

Meldal, B. H. M., Perfetto, L., Combe, C., Lubiana, T., Ferreira Cavalcante, J. V., Bye-A-Jee, H., Waagmeester, A., del-Toro, N., Shrivastava, A., Barrera, E., Wong, E., Mlecnik, B., Bindea, G., Panneerselvam, K., **Willighagen, E.**, Rappsilber, J., Porras, P., Hermjakob, H., & Orchard, S. (2021). Complex Portal 2022: new curation frontiers. *Nucleic Acids Research*, *50*(D1), D578–D586. doi:[10.1093/nar/gkab991](https://doi.org/10.1093/nar/gkab991)

Hanspers, K., Kutmon, M., Coort, S. L., Digles, D., Dupuis, L. J., Ehrhart, F., Hu, F., Lopes, E. N., Martens, M., Pham, N., Shin, W., Slenter, D. N., Waagmeester, A., **Willighagen, E. L.**, Winckers, L. A., Evelo, C. T., & Pico, A. R. (2021). Ten simple rules for creating reusable pathway models for computational analysis and visualization. *PLOS Computational Biology*, *17*(8), e1009226. doi:[10.1371/journal.pcbi.1009226](https://doi.org/10.1371/journal.pcbi.1009226)

Kyle, J. E., Aimo, L., Bridge, A. J., Clair, G., Fedorova, M., Helms, J. B., Molenaar, M. R., Ni, Z., Orešič, M., Slenter, D., **Willighagen, E.**, & Webb-Robertson, B.-J. M. (2021). Interpreting the lipidome: bioinformatic approaches to embrace the complexity. *Metabolomics*, *17*(6). doi:[10.1007/s11306-021-01802-6](https://doi.org/10.1007/s11306-021-01802-6)

**Willighagen, E.** (2021). Bacting: a next generation, command line version of Bioclipse. *Journal of Open Source Software*, *6*(62), 2558. doi:[10.21105/joss.02558](https://doi.org/10.21105/joss.02558)

Winckers, L. A., Evelo, C. T., **Willighagen, E. L.**, & Kutmon, M. (2021). Investigating the Molecular Processes behind the Cell-Specific Toxicity Response to Titanium Dioxide Nanobelts. *International Journal of Molecular Sciences*, *22*(17), 9432. doi:[10.3390/ijms22179432](https://doi.org/10.3390/ijms22179432)

Ehrhart, F., Jacobsen, A., Rigau, M., Bosio, M., Kaliyaperumal, R., Laros, J. F. J., **Willighagen, E. L.**, Valencia, A., Roos, M., Capella-Gutierrez, S., Curfs, L. M. G., & Evelo, C. T. (2021). A catalogue of 863 Rett-syndrome-causing MECP2 mutations and lessons learned from data integration. *Scientific Data*, *8*(1). doi:[10.1038/s41597-020-00794-7](https://doi.org/10.1038/s41597-020-00794-7)

Ehrhart, F., **Willighagen, E. L.**, Kutmon, M., van Hoften, M., Curfs, L. M. G., & Evelo, C. T. (2021). A resource to explore the discovery of rare diseases and their causative genes. *Scientific Data*, *8*(1). doi:[10.1038/s41597-021-00905-y](https://doi.org/10.1038/s41597-021-00905-y)

Murugadoss, S. (2021). A strategy towards the generation of testable adverse outcome pathways for nanomaterials. *ALTEX*. doi:[10.14573/altex.2102191](https://doi.org/10.14573/altex.2102191)

Waagmeester, A., **Willighagen, E. L.**, Su, A. I., Kutmon, M., Gayo, J. E. L., Fernández-Álvarez, D., Groom, Q., Schaap, P. J., Verhagen, L. M., & Koehorst, J. J. (2021). A protocol for adding knowledge to Wikidata: aligning resources on human coronaviruses. *BMC Biology*, *19*(1). doi:[10.1186/s12915-020-00940-y](https://doi.org/10.1186/s12915-020-00940-y)

`2020`
Ammar, A., Bonaretti, S., Winckers, L., Quik, J., Bakker, M., Maier, D., Lynch, I., van Rijn, J., & **Willighagen, E.** (2020). A Semi-Automated Workflow for FAIR Maturity Indicators in the Life Sciences. *Nanomaterials*, *10*(10), 2068. doi:[10.3390/nano10102068](https://doi.org/10.3390/nano10102068)

Lynch, I., Afantitis, A., Exner, T., Himly, M., Lobaskin, V., Doganis, P., Maier, D., Sanabria, N., Papadiamantis, A. G., Rybinska-Fryca, A., Gromelski, M., Puzyn, T., **Willighagen, E.**, Johnston, B. D., Gulumian, M., Matzke, M., Green Etxabe, A., Bossa, N., Serra, A., … Melagraki, G. (2020). Can an InChI for Nano Address the Need for a Simplified Representation of Complex Nanomaterials across Experimental and Nanoinformatics Studies? *Nanomaterials*, *10*(12), 2493. doi:[10.3390/nano10122493](https://doi.org/10.3390/nano10122493)

Martens, M., Ammar, A., Riutta, A., Waagmeester, A., Slenter, D. N., Hanspers, K., A. Miller, R., Digles, D., Lopes, E. N., Ehrhart, F., Dupuis, L. J., Winckers, L. A., Coort, S. L., **Willighagen, E. L.**, Evelo, C. T., Pico, A. R., & Kutmon, M. (2020). WikiPathways: connecting communities. *Nucleic Acids Research*, *49*(D1), D613–D621. doi:[10.1093/nar/gkaa1024](https://doi.org/10.1093/nar/gkaa1024)

Afantitis, A., Melagraki, G., Isigonis, P., Tsoumanis, A., Varsou, D. D., Valsami-Jones, E., Papadiamantis, A., Ellis, L.-J. A., Sarimveis, H., Doganis, P., Karatzas, P., Tsiros, P., Liampa, I., Lobaskin, V., Greco, D., Serra, A., Kinaret, P. A. S., Saarimäki, L. A., Grafström, R., … Lynch, I. (2020). NanoSolveIT Project: Driving nanoinformatics research to develop innovative and integrated tools for in silico nanosafety assessment. *Computational and Structural Biotechnology Journal*, *18*, 583–602. doi:[10.1016/j.csbj.2020.02.023](https://doi.org/10.1016/j.csbj.2020.02.023)

Jacobsen, A., de Miranda Azevedo, R., Juty, N., Batista, D., Coles, S., Cornet, R., Courtot, M., Crosas, M., Dumontier, M., Evelo, C. T., Goble, C., Guizzardi, G., Hansen, K. K., Hasnain, A., Hettne, K., Heringa, J., Hooft, R. W. W., Imming, M., Jeffery, K. G., … Schultes, E. (2020). FAIR Principles: Interpretations and Implementation Considerations. *Data Intelligence*, *2*(1–2), 10–29. doi:[10.1162/dint_r_00024](https://doi.org/10.1162/dint_r_00024)

Ostaszewski, M., Mazein, A., Gillespie, M. E., Kuperstein, I., Niarakis, A., Hermjakob, H., Pico, A. R., **Willighagen, E. L.**, Evelo, C. T., Hasenauer, J., Schreiber, F., Dräger, A., Demir, E., Wolkenhauer, O., Furlong, L. I., Barillot, E., Dopazo, J., Orta-Resendiz, A., Messina, F., … Schneider, R. (2020). COVID-19 Disease Map, building a computational repository of SARS-CoV-2 virus-host interaction mechanisms. *Scientific Data*, *7*(1). doi:[10.1038/s41597-020-0477-8](https://doi.org/10.1038/s41597-020-0477-8)

Waagmeester, A., Stupp, G., Burgstaller-Muehlbacher, S., Good, B. M., Griffith, M., Griffith, O. L., Hanspers, K., Hermjakob, H., Hudson, T. S., Hybiske, K., Keating, S. M., Manske, M., Mayers, M., Mietchen, D., Mitraka, E., Pico, A. R., Putman, T., Riutta, A., Queralt-Rosinach, N., … Su, A. I. (2020). Wikidata as a knowledge graph for the life sciences. *eLife*, *9*. doi:[10.7554/elife.52614](https://doi.org/10.7554/elife.52614)

`2019`
Stanstrup, J., Broeckling, C., Helmus, R., Hoffmann, N., Mathé, E., Naake, T., Nicolotti, L., Peters, K., Rainer, J., Salek, R., Schulze, T., Schymanski, E., Stravs, M., Thévenot, E., Treutler, H., Weber, R., **Willighagen, E.**, Witting, M., & Neumann, S. (2019). The metaRbolomics Toolbox in Bioconductor and beyond. *Metabolites*, *9*(10), 200. doi:[10.3390/metabo9100200](https://doi.org/10.3390/metabo9100200)

Kutmon, M., Ehrhart, F., **Willighagen, E. L.**, Evelo, C. T., & Coort, S. L. (2019). CyTargetLinker app update: A flexible solution for network extension in Cytoscape. *F1000Research*, *7*, 743. doi:[10.12688/f1000research.14613.2](https://doi.org/10.12688/f1000research.14613.2)

Menden, M. P., Wang, D., Mason, M. J., Szalai, B., Bulusu, K. C., Guan, Y., Yu, T., Kang, J., Jeon, M., Wolfinger, R., Nguyen, T., Zaslavskiy, M., Abante, J., Abecassis, B. S., Aben, N., Aghamirzaie, D., Aittokallio, T., Akhtari, F. S., Al-lazikani, B., … Saez-Rodriguez, J. (2019). Community assessment to advance computational prediction of cancer drug combinations in a pharmacogenomic screen. *Nature Communications*, *10*(1). doi:[10.1038/s41467-019-09799-2](https://doi.org/10.1038/s41467-019-09799-2)

`2018`
Miller, R. A., Ehrhart, F., Eijssen, L. M. T., Slenter, D. N., Curfs, L. M. G., Evelo, C. T., **Willighagen, E. L.**, & Kutmon, M. (2019). Beyond Pathway Analysis: Identification of Active Subnetworks in Rett Syndrome. *Frontiers in Genetics*, *10*. doi:[10.3389/fgene.2019.00059](https://doi.org/10.3389/fgene.2019.00059)

Townend, G. S., Ehrhart, F., van Kranen, H. J., Wilkinson, M., Jacobsen, A., Roos, M., **Willighagen, E. L.**, van Enckevort, D., Evelo, C. T., & Curfs, L. M. G. (2018). *MECP2*
 variation in Rett syndrome-An overview of current coverage of genetic and phenotype data within existing databases. *Human Mutation*, *39*(7), 914–924. doi:[10.1002/humu.23542](https://doi.org/10.1002/humu.23542)

Martens, M., Verbruggen, T., Nymark, P., Grafström, R., Burgoon, L. D., Aladjov, H., Torres Andón, F., Evelo, C. T., & **Willighagen, E. L.** (2018). Introducing WikiPathways as a Data-Source to Support Adverse Outcome Pathways for Regulatory Risk Assessment of Chemicals and Nanomaterials. *Frontiers in Genetics*, *9*. doi:[10.3389/fgene.2018.00661](https://doi.org/10.3389/fgene.2018.00661)

Miller, R. A., Woollard, P., **Willighagen, E. L.**, Digles, D., Kutmon, M., Loizou, A., Waagmeester, A., Senger, S., & Evelo, C. T. (2018). Explicit interaction information from WikiPathways in RDF facilitates drug discovery in the Open PHACTS Discovery Platform. *F1000Research*, *7*, 75. doi:[10.12688/f1000research.13197.2](https://doi.org/10.12688/f1000research.13197.2)

Slenter, D. N., Kutmon, M., Hanspers, K., Riutta, A., Windsor, J., Nunes, N., Mélius, J., Cirillo, E., Coort, S. L., Digles, D., Ehrhart, F., Giesbertz, P., Kalafati, M., Martens, M., Miller, R., Nishida, K., Rieswijk, L., Waagmeester, A., Eijssen, L. M. T., … **Willighagen, E. L.** (2017). WikiPathways: a multifaceted pathway database bridging metabolomics to other omics research. *Nucleic Acids Research*, *46*(D1), D661–D667. doi:[10.1093/nar/gkx1064](https://doi.org/10.1093/nar/gkx1064)

Karcher, S., **Willighagen, E. L.**, Rumble, J., Ehrhart, F., Evelo, C. T., Fritts, M., Gaheen, S., Harper, S. L., Hoover, M. D., Jeliazkova, N., Lewinski, N., Marchese Robinson, R. L., Mills, K. C., Mustad, A. P., Thomas, D. G., Tsiliki, G., & Hendren, C. O. (2018). Integration among databases and data sets to support productive nanotechnology: Challenges and recommendations. *NanoImpact*, *9*, 85–101. doi:[10.1016/j.impact.2017.11.002](https://doi.org/10.1016/j.impact.2017.11.002)

`2017`
Lampa, S., **Willighagen, E.**, Kohonen, P., King, A., Vrandečić, D., Grafström, R., & Spjuth, O. (2017). RDFIO: extending Semantic MediaWiki for interoperable biomedical data management. *Journal of Biomedical Semantics*, *8*(1). doi:[10.1186/s13326-017-0136-y](https://doi.org/10.1186/s13326-017-0136-y)

Leist, M., Ghallab, A., Graepel, R., Marchan, R., Hassan, R., Bennekou, S. H., Limonciel, A., Vinken, M., Schildknecht, S., Waldmann, T., Danen, E., van Ravenzwaay, B., Kamp, H., Gardner, I., Godoy, P., Bois, F. Y., Braeuning, A., Reif, R., Oesch, F., … Hengstler, J. G. (2017). Adverse outcome pathways: opportunities, limitations and open questions. *Archives of Toxicology*, *91*(11), 3477–3505. doi:[10.1007/s00204-017-2045-3](https://doi.org/10.1007/s00204-017-2045-3)

Nymark, P., Rieswijk, L., Ehrhart, F., Jeliazkova, N., Tsiliki, G., Sarimveis, H., Evelo, C. T., Hongisto, V., Kohonen, P., **Willighagen, E.**, & Grafström, R. C. (2017). A Data Fusion Pipeline for Generating and Enriching Adverse Outcome Pathway Descriptions. *Toxicological Sciences*, *162*(1), 264–275. doi:[10.1093/toxsci/kfx252](https://doi.org/10.1093/toxsci/kfx252)

van Rijswijk, M., Beirnaert, C., Caron, C., Cascante, M., Dominguez, V., Dunn, W. B., Ebbels, T. M. D., Giacomoni, F., Gonzalez-Beltran, A., Hankemeier, T., Haug, K., Izquierdo-Garcia, J. L., Jimenez, R. C., Jourdan, F., Kale, N., Klapa, M. I., Kohlbacher, O., Koort, K., Kultima, K., … Steinbeck, C. (2017). The future of metabolomics in ELIXIR. *F1000Research*, *6*, 1649. doi:[10.12688/f1000research.12342.2](https://doi.org/10.12688/f1000research.12342.2)

Kohonen, P., Parkkinen, J. A., **Willighagen, E. L.**, Ceder, R., Wennerberg, K., Kaski, S., & Grafström, R. C. (2017). A transcriptomics data-driven gene space accurately predicts liver cytopathology and drug-induced liver injury. *Nature Communications*, *8*(1). doi:[10.1038/ncomms15932](https://doi.org/10.1038/ncomms15932)

**Willighagen, E. L.**, Mayfield, J. W., Alvarsson, J., Berg, A., Carlsson, L., Jeliazkova, N., Kuhn, S., Pluskal, T., Rojas-Chertó, M., Spjuth, O., Torrance, G., Evelo, C. T., Guha, R., & Steinbeck, C. (2017). The Chemistry Development Kit (CDK) v2.0: atom typing, depiction, molecular formulas, and substructure searching. *Journal of Cheminformatics*, *9*(1). doi:[10.1186/s13321-017-0220-4](https://doi.org/10.1186/s13321-017-0220-4)

`2016`
Wohlgemuth, G., Mehta, S. S., Mejia, R. F., Neumann, S., Pedrosa, D., Pluskal, T., Schymanski, E. L., **Willighagen, E. L.**, Wilson, M., Wishart, D. S., Arita, M., Dorrestein, P. C., Bandeira, N., Wang, M., Schulze, T., Salek, R. M., Steinbeck, C., Nainala, V. C., Mistrik, R., … Fiehn, O. (2016). SPLASH, a hashed identifier for mass spectra. *Nature Biotechnology*, *34*(11), 1099–1101. doi:[10.1038/nbt.3689](https://doi.org/10.1038/nbt.3689)

Spjuth, O., Rydberg, P., **Willighagen, E. L.**, Evelo, C. T., & Jeliazkova, N. (2016). XMetDB: an open access database for xenobiotic metabolism. *Journal of Cheminformatics*, *8*(1). doi:[10.1186/s13321-016-0161-3](https://doi.org/10.1186/s13321-016-0161-3)

Waagmeester, A., Kutmon, M., Riutta, A., Miller, R., **Willighagen, E. L.**, Evelo, C. T., & Pico, A. R. (2016). Using the Semantic Web for Rapid Integration of WikiPathways with Other Biological Online Data Resources. *PLOS Computational Biology*, *12*(6), e1004989. doi:[10.1371/journal.pcbi.1004989](https://doi.org/10.1371/journal.pcbi.1004989)

`2015`
Kutmon, M., Riutta, A., Nunes, N., Hanspers, K., **Willighagen, E. L.**, Bohler, A., Mélius, J., Waagmeester, A., Sinha, S. R., Miller, R., Coort, S. L., Cirillo, E., Smeets, B., Evelo, C. T., & Pico, A. R. (2015). WikiPathways: capturing the full diversity of pathway knowledge. *Nucleic Acids Research*, *44*(D1), D488–D494. doi:[10.1093/nar/gkv1024](https://doi.org/10.1093/nar/gkv1024)

Tsiliki, G., Munteanu, C. R., Seoane, J. A., Fernandez-Lozano, C., Sarimveis, H., & **Willighagen, E. L.** (2015). RRegrs: an R package for computer-aided model selection with multiple regression models. *Journal of Cheminformatics*, *7*(1). doi:[10.1186/s13321-015-0094-2](https://doi.org/10.1186/s13321-015-0094-2)

Bohler, A., Eijssen, L. M. T., van Iersel, M. P., Leemans, C., **Willighagen, E. L.**, Kutmon, M., Jaillard, M., & Evelo, C. T. (2015). Automatically visualise and analyse data on pathways using PathVisioRPC from any programming environment. *BMC Bioinformatics*, *16*(1). doi:[10.1186/s12859-015-0708-8](https://doi.org/10.1186/s12859-015-0708-8)

Fu, G., Batchelor, C., Dumontier, M., Hastings, J., **Willighagen, E.**, & Bolton, E. (2015). PubChemRDF: towards the semantic annotation of PubChem compound and substance databases. *Journal of Cheminformatics*, *7*(1). doi:[10.1186/s13321-015-0084-4](https://doi.org/10.1186/s13321-015-0084-4)

Hastings, J., Jeliazkova, N., Owen, G., Tsiliki, G., Munteanu, C. R., Steinbeck, C., & **Willighagen, E.** (2015). eNanoMapper: harnessing ontologies to enable data integration for nanomaterial risk assessment. *Journal of Biomedical Semantics*, *6*(1). doi:[10.1186/s13326-015-0005-5](https://doi.org/10.1186/s13326-015-0005-5)

Jeliazkova, N., Chomenidis, C., Doganis, P., Fadeel, B., Grafström, R., Hardy, B., Hastings, J., Hegi, M., Jeliazkov, V., Kochev, N., Kohonen, P., Munteanu, C. R., Sarimveis, H., Smeets, B., Sopasakis, P., Tsiliki, G., Vorgrimmler, D., & **Willighagen, E.** (2015). The eNanoMapper database for nanomaterial safety information. *Beilstein Journal of Nanotechnology*, *6*, 1609–1634. doi:[10.3762/bjnano.6.165](https://doi.org/10.3762/bjnano.6.165)

`2014`
Grafström, R. C., Nymark, P., Hongisto, V., Spjuth, O., Ceder, R., **Willighagen, E.**, Hardy, B., Kaski, S., & Kohonen, P. (2015). Toward the Replacement of Animal Experiments through the Bioinformatics-driven Analysis of “Omics” Data from Human Cell Cultures. Alternatives to Laboratory Animals, 43(5), 325–332. doi:[10.1177/02611929150430050](https://doi.org/10.1177/02611929150430050)

`2013`
Spjuth, O., Berg, A., Adams, S., & **Willighagen, E. L.** (2013). Applications of the InChI in cheminformatics with the CDK and Bioclipse. *Journal of Cheminformatics*, *5*(1). doi:[10.1186/1758-2946-5-14](https://doi.org/10.1186/1758-2946-5-14)

**Willighagen, E. L.**, Waagmeester, A., Spjuth, O., Ansell, P., Williams, A. J., Tkachenko, V., Hastings, J., Chen, B., & Wild, D. J. (2013). The ChEMBL database as linked open data. *Journal of Cheminformatics*, *5*(1). doi:[10.1186/1758-2946-5-23](https://doi.org/10.1186/1758-2946-5-23)

`2012`
Spjuth, O., Georgiev, V., Carlsson, L., Alvarsson, J., Berg, A., **Willighagen, E.**, Wikberg, J. E. S., & Eklund, M. (2012). Bioclipse-R: integrating management and visualization of life science data with statistical analysis. *Bioinformatics*, *29*(2), 286–289. doi:[10.1093/bioinformatics/bts681](https://doi.org/10.1093/bioinformatics/bts681)

Kohonen, P., Benfenati, E., Bower, D., Ceder, R., Crump, M., Cross, K., Grafström, R. C., Healy, L., Helma, C., Jeliazkova, N., Jeliazkov, V., Maggioni, S., Miller, S., Myatt, G., Rautenberg, M., Stacey, G., **Willighagen, E.**, Wiseman, J., & Hardy, B. (2013). The ToxBank Data Warehouse: Supporting the Replacement of In Vivo Repeated Dose Systemic Toxicity Testing. *Molecular Informatics*, *32*(1), 47–63. doi:[10.1002/minf.201200114](https://doi.org/10.1002/minf.201200114)

Marshall, M. S., Boyce, R., Deus, H. F., Zhao, J., **Willighagen, E. L.**, Samwald, M., Pichler, E., Hajagos, J., Prud’hommeaux, E., & Stephens, S. (2012). Emerging practices for mapping and linking life sciences data using RDF — A case series. *Journal of Web Semantics*, *14*, 2–13. doi:[10.1016/j.websem.2012.02.003](https://doi.org/10.1016/j.websem.2012.02.003)

Guha, R., & **Willighagen, E**. (2012). A Survey of Quantitative Descriptions of Molecular Structure. *Current Topics in Medicinal Chemistry*, *12*(18), 1946–1956. doi:[10.2174/156802612804910278](https://doi.org/10.2174/156802612804910278)

Neylon, C., Aerts, J., Brown, C. T., Coles, S. J., Hatton, L., Lemire, D., Millman, K. J., Murray-Rust, P., Perez, F., Saunders, N., Shah, N., Smith, A., Varoquaux, G., & **Willighagen, E.** (2012). Changing computational research. The challenges ahead. *Source Code for Biology and Medicine*, *7*(1). doi:[10.1186/1751-0473-7-2](https://doi.org/10.1186/1751-0473-7-2)

Williams, A. J., Ekins, S., Spjuth, O., & **Willighagen, E. L.** (2012). Accessing, Using, and Creating Chemical Property Databases for Computational Toxicology Modeling. In *Methods in Molecular Biology* (pp. 221–241). Humana Press. doi:[10.1007/978-1-62703-050-2_10](https://doi.org/10.1007/978-1-62703-050-2_10)

Spjuth, O., Carlsson, L., Alvarsson, J., Georgiev, V., **Willighagen, E.**, & Eklund, M. (2012). Open Source Drug Discovery with Bioclipse. *Current Topics in Medicinal Chemistry*, *12*(18), 1980–1986. doi:[10.2174/156802612804910287](https://doi.org/10.2174/156802612804910287)

Wegner, J. K., Sterling, A., Guha, R., Bender, A., Faulon, J.-L., Hastings, J., O’Boyle, N., Overington, J., Van Vlijmen, H., & **Willighagen, E.** (2012). Cheminformatics. *Communications of the ACM*, *55*(11), 65–75. doi:[10.1145/2366316.2366334](https://doi.org/10.1145/2366316.2366334)

Williams, A. J., Harland, L., Groth, P., Pettifer, S., Chichester, C., **Willighagen, E. L.**, Evelo, C. T., Blomberg, N., Ecker, G., Goble, C., & Mons, B. (2012). Open PHACTS: semantic interoperability for drug discovery. *Drug Discovery Today*, *17*(21–22), 1188–1198. doi:[10.1016/j.drudis.2012.05.016](https://doi.org/10.1016/j.drudis.2012.05.016)

`2011`
Hastings, J., Chepelev, L., **Willighagen, E.**, Adams, N., Steinbeck, C., & Dumontier, M. (2011). The Chemical Information Ontology: Provenance and Disambiguation for Chemical Data on the Biological Semantic Web. *PLoS ONE*, *6*(10), e25513. doi:[10.1371/journal.pone.0025513](https://doi.org/10.1371/journal.pone.0025513)

Rojas-Chertó, M., Kasper, P. T., **Willighagen, E. L.**, Vreeken, R. J., Hankemeier, T., & Reijmers, T. H. (2011). Elemental composition determination based on MS*n*. *Bioinformatics*, *27*(17), 2376–2383. doi:[10.1093/bioinformatics/btr409](https://doi.org/10.1093/bioinformatics/btr409)

Samwald, M., Jentzsch, A., Bouton, C., Kallesøe, C. S., **Willighagen, E.**, Hajagos, J., Marshall, M. S., Prud’hommeaux, E., Hassanzadeh, O., Pichler, E., & Stephens, S. (2011). Linked open drug data for pharmaceutical research and development. *Journal of Cheminformatics*, *3*(1). doi:[10.1186/1758-2946-3-19](https://doi.org/10.1186/1758-2946-3-19)

Truszkowski, A., Neumann, S., Zielesny, A., **Willighagen, E.**, & Steinbeck, C. (2011). CDK-Taverna 2.0: migration and enhancements of an open-source pipelining solution. *Journal of Cheminformatics*, *3*(S1). doi:[10.1186/1758-2946-3-s1-p5](https://doi.org/10.1186/1758-2946-3-s1-p5)

**Willighagen, E. L.**, Jeliazkova, N., Hardy, B., Grafström, R. C., & Spjuth, O. (2011). Computational toxicology using the OpenTox application programming interface and Bioclipse. *BMC Research Notes*, *4*(1). doi:[10.1186/1756-0500-4-487](https://doi.org/10.1186/1756-0500-4-487)

Jessop, D. M., Adams, S. E., **Willighagen, E. L.**, Hawizy, L., & Murray-Rust, P. (2011). OSCAR4: a flexible architecture for chemical text-mining. *Journal of Cheminformatics*, *3*(1). doi:[10.1186/1758-2946-3-41](https://doi.org/10.1186/1758-2946-3-41)

O’Boyle, N. M., Guha, R., **Willighagen, E. L.**, Adams, S. E., Alvarsson, J., Bradley, J.-C., Filippov, I. V., Hanson, R. M., Hanwell, M. D., Hutchison, G. R., James, C. A., Jeliazkova, N., Lang, A. S., Langner, K. M., Lonie, D. C., Lowe, D. M., Pansanel, J., Pavlov, D., Spjuth, O., … Murray-Rust, P. (2011). Open Data, Open Source and Open Standards in chemistry: The Blue Obelisk five years on. *Journal of Cheminformatics*, *3*(1). doi:[10.1186/1758-2946-3-37](https://doi.org/10.1186/1758-2946-3-37)

Truszkowski, A., Jayaseelan, K. V., Neumann, S., **Willighagen, E. L.**, Zielesny, A., & Steinbeck, C. (2011). New developments on the cheminformatics open workflow environment CDK-Taverna. *Journal of Cheminformatics*, *3*(1). doi:[10.1186/1758-2946-3-54](https://doi.org/10.1186/1758-2946-3-54)

**Willighagen, E. L.**, Alvarsson, J., Andersson, A., Eklund, M., Lampa, S., Lapins, M., Spjuth, O., & Wikberg, J. E. (2011). Linking the Resource Description Framework to cheminformatics and proteochemometrics. *Journal of Biomedical Semantics*, *2*(Suppl 1), S6. doi:[10.1186/2041-1480-2-s1-s6](https://doi.org/10.1186/2041-1480-2-s1-s6)

**Willighagen, E. L.**, & Brändle, M. P. (2011). Resource description framework technologies in chemistry. *Journal of Cheminformatics*, *3*(1). doi:[10.1186/1758-2946-3-15](https://doi.org/10.1186/1758-2946-3-15)

`2010`
Kuhn, T., **Willighagen, E. L.**, Zielesny, A., & Steinbeck, C. (2010). CDK-Taverna: an open workflow environment for cheminformatics. *BMC Bioinformatics*, *11*(1). doi:[10.1186/1471-2105-11-159](https://doi.org/10.1186/1471-2105-11-159)

Spjuth, O., **Willighagen, E. L.**, Guha, R., Eklund, M., & Wikberg, J. E. (2010). Towards interoperable and reproducible QSAR analyses: Exchange of datasets. *Journal of Cheminformatics*, *2*(1). doi:[10.1186/1758-2946-2-5](https://doi.org/10.1186/1758-2946-2-5)

Wohlgemuth, G., Haldiya, P. K., **Willighagen, E.**, Kind, T., & Fiehn, O. (2010). The Chemical Translation Service—a web-based tool to improve standardization of metabolomic reports. *Bioinformatics*, *26*(20), 2647–2648. doi:[10.1093/bioinformatics/btq476](https://doi.org/10.1093/bioinformatics/btq476)

`2009`
Spjuth, O., Alvarsson, J., Berg, A., Eklund, M., Kuhn, S., Mäsak, C., Torrance, G., Wagener, J., **Willighagen, E. L.**, Steinbeck, C., & Wikberg, J. E. (2009). Bioclipse 2: A scriptable integration platform for the life sciences. *BMC Bioinformatics*, *10*(1). doi:[10.1186/1471-2105-10-397](https://doi.org/10.1186/1471-2105-10-397)

Wagener, J., Spjuth, O., **Willighagen, E. L.**, & Wikberg, J. E. (2009). XMPP for cloud computing in bioinformatics supporting discovery and invocation of asynchronous web services. *BMC Bioinformatics*, *10*(1). doi:[10.1186/1471-2105-10-279](https://doi.org/10.1186/1471-2105-10-279)

`2007`
**Willighagen, E. L.**, Wehrens, R., Melssen, W., de Gelder, R., & Buydens, L. M. C. (2007). Supervised Self-Organizing Maps in Crystal Property and Structure Prediction. *Crystal Growth &amp; Design*, *7*(9), 1738–1745. doi:[10.1021/cg060872y](https://doi.org/10.1021/cg060872y)

**Willighagen, E. L.**, O’Boyle, N. M., Gopalakrishnan, H., Jiao, D., Guha, R., Steinbeck, C., & Wild, D. J. (2007). Userscripts for the Life Sciences. *BMC Bioinformatics*, *8*(1). doi:[10.1186/1471-2105-8-487](https://doi.org/10.1186/1471-2105-8-487)

Kuhn, S., Helmus, T., Lancashire, R. J., Murray-Rust, P., Rzepa, H. S., Steinbeck, C., & **Willighagen, E. L.** (2007). Chemical Markup, XML, and the World Wide Web. 7. CMLSpect, an XML Vocabulary for Spectral Data. *Journal of Chemical Information and Modeling*, *47*(6), 2015–2034. doi:[10.1021/ci600531a](https://doi.org/10.1021/ci600531a)

Spjuth, O., Helmus, T., **Willighagen, E. L.**, Kuhn, S., Eklund, M., Wagener, J., Murray-Rust, P., Steinbeck, C., & Wikberg, J. E. (2007). Bioclipse: an open source workbench for chemo- and bioinformatics. *BMC Bioinformatics*, *8*(1). doi:[10.1186/1471-2105-8-59](https://doi.org/10.1186/1471-2105-8-59)

`2006`
Guha, R., Howard, M. T., Hutchison, G. R., Murray-Rust, P., Rzepa, H., Steinbeck, C., Wegner, J., & **Willighagen, E. L.** (2006). The Blue Obelisk—Interoperability in Chemical Informatics. *Journal of Chemical Information and Modeling*, *46*(3), 991–998. doi:[10.1021/ci050400b](https://doi.org/10.1021/ci050400b)

Steinbeck, C., Hoppe, C., Kuhn, S., Floris, M., Guha, R., & **Willighagen, E.** (2006). Recent Developments of the Chemistry Development Kit (CDK) - An Open-Source Java Library for Chemo- and Bioinformatics. *Current Pharmaceutical Design*, *12*(17), 2111–2120. doi:[10.2174/138161206777585274](https://doi.org/10.2174/138161206777585274)

**Willighagen, E. L.**, Denissen, H. M. G. W., Wehrens, R., & Buydens, L. M. C. (2006). On the Use of 1H and 13C 1D NMR Spectra as QSPR Descriptors. *Journal of Chemical Information and Modeling*, *46*(2), 487–494. doi:[10.1021/ci050282s](https://doi.org/10.1021/ci050282s)

**Willighagen, E. L.**, Wehrens, R., & Buydens, L. M. C. (2006). Molecular Chemometrics. *Critical Reviews in Analytical Chemistry*, *36*(3–4), 189–198. doi:[10.1080/10408340600969601](https://doi.org/10.1080/10408340600969601)

`2005`
**Willighagen, E. L.**, Wehrens, R., Verwer, P., de Gelder, R., & Buydens, L. M. C. (2005). Method for the computational comparison of crystal structures. *Acta Crystallographica Section B Structural Science*, *61*(1), 29–36. doi:[10.1107/s0108768104028344](https://doi.org/10.1107/s0108768104028344)

`2004`
Murray-Rust, P., Rzepa, H. S., Williamson, M. J., & **Willighagen, E. L.** (2004). Chemical Markup, XML, and the World Wide Web. 5. Applications of Chemical Metadata in RSS Aggregators. *Journal of Chemical Information and Computer Sciences*, *44*(2), 462–469. doi:[10.1021/ci034244p](https://doi.org/10.1021/ci034244p)

`2003`
Steinbeck, C., Han, Y., Kuhn, S., Horlacher, O., Luttmann, E., & **Willighagen, E**. (2003). The Chemistry Development Kit (CDK):  An Open-Source Java Library for Chemo- and Bioinformatics. *Journal of Chemical Information and Computer Sciences*, *43*(2), 493–500. doi:[10.1021/ci025584y](https://doi.org/10.1021/ci025584y)

`2002`
Buijnsters, P. J. J. A., García Rodríguez, C. L., **Willighagen, E. L.**, Sommerdijk, N. A. J. M., Kremer, A., Camilleri, P., Feiters, M. C., Nolte, R. J. M., & Zwanenburg, B. (2002). Cationic Gemini Surfactants Based on Tartaric Acid: Synthesis, Aggregation, Monolayer Behaviour, and Interaction with DNA. *European Journal of Organic Chemistry*, *2002*(8), 1397–1406. doi:[10.1002/1099-0690(200204)2002:8<1397::aid-ejoc1397>3.0.co;2-6](https://doi.org/10.1002/1099-0690(200204)2002:8<1397::aid-ejoc1397>3.0.co;2-6)

`2001`
**Willighagen, E.**, (2001). Processing CML Conventions in Java, *Internet Journal of Chemistry*, *4*.

`2000`
Krause, S., **Willighagen, E.**, & Steinbeck, C. (2000). JChemPaint - Using the Collaborative Forces of the Internet to Develop a Free Editor for 2D Chemical Structures. *Molecules*, *5*(1), 93–98. doi:[10.3390/50100093](https://doi.org/10.3390/50100093)

### Books and chapters

Scientific publication as a book or as a contribution to a book including proceedings.

### Books

**E. Willighagen**, Groovy Cheminformatics with the Chemistry Development Kit, Lulu.com, 2011 (now as [free and open access eBook](https://cdk.github.io/cdkbook/)).

J. Wikberg, M. Eklund, **E. Willighagen**, O. Spjuth, M. Lapins, O. Engkvist,
J. Alvarsson, Introduction to Pharmaceutical Bioinformatics. Oakleaf Academic, Stockholm, Sweden. ISBN: 9-789-19794030-6, 2010.

### Conference proceedings

Kuhn, T., Banda, J. M., **Willighagen, E.**, Ehrhart, F., Evelo, C., Malas, T. B., Dumontier, M., Merono-Penuela, A., Malic, A., Poelen, J. H., Hurlbert, A. H., Centeno Ortiz, E., Furlong, L. I., Queralt-Rosinach, N., & Chichester, C. (2018). Nanopublications: A Growing Resource of Provenance-Centric Scientific Linked Data. *2018 IEEE 14th International Conference on E-Science (e-Science)*, 83–92. doi:[10.1109/escience.2018.00024](https://doi.org/10.1109/escience.2018.00024)

T. Kuhn, **E. Willighagen**, C. Evelo, N. Queralt-Rosinach, E. Centeno, L.I. Furlong
LI. *Reliable Granular References to Changing Linked Data*. In: d’Amato C, Fernandez
M, Tamma V, Lecue F, Cudré-Mauroux P, Sequeda J, et al., editors. The Semantic
Web – ISWC 2017. Cham: Springer International Publishing; 2017. p.
436–51. doi:[10.1007/978-3-319-68288-4_26](https://doi.org/10.1007/978-3-319-68288-4_26)

Nielsen, F. Å., Mietchen, D., & **Willighagen, E.** (2017). Scholia, Scientometrics and Wikidata. In *Lecture Notes in Computer Science* (pp. 237–259). Springer International Publishing. doi:[10.1007/978-3-319-70407-4_36](https://doi.org/10.1007/978-3-319-70407-4_36)

C. Batchelor, C. Brenninkmeijer, C. Chichester, M. Davies, D. Digles, I. Dunlop,
C. Evelo, A. Gaulton, C. Goble, A. Gray, P. Groth, L. Harland, K. Karapetyan, A.
Loizou, J. Overington, S. Pettifer, J. Steele, R. Stevens, V. Tkachenko, A. Waag-
meester, A. Williams, **E. Willighagen**, *Scientific lenses to support multiple views
over linked chemistry data*. In: Mika, P., Tudorache, T., Bernstein, A., Welty, C.,
Knoblock, C., Vrandečić, D., Groth, P., Noy, N., Janowicz, K., Goble, C. (Eds.), The
Semantic Web – ISWC 2014. Vol. 8796 of Lecture Notes in Computer Science.
Springer International Publishing, Cham, Ch. 7, pp. 98-113, 2014. doi:[10.1007/978-3-319-11964-9_7](https://doi.org/10.1007/978-3-319-11964-9_7)

N. Jeliazkova, P. Doganis, B. Fadeel, R. Grafstrom, J. Hastings, V. Jeliazkov, P.
Kohonen, C. Munteanu, H. Sarimveis, B. Smeets, G. Tsiliki, D. Vorgrimmler,
**E. Willighagen**, *The first eNanoMapper prototype: A substance database to support
safe-by-design*. In: Bioinformatics and Biomedicine (BIBM), 2014 IEEE Interna-
tional Conference on. IEEE, pp. 1-9. doi:[10.1109/bibm.2014.6999367](https://doi.org/10.1109/bibm.2014.6999367), 2014.

J. Hastings, **E. Willighagen**, G. Owen, N. Jeliazkova, The eNanoMapper Consortium,
C. Steinbeck, *eNanoMapper: Opportunities and challenges in using ontologies
to enable data integration for nanomaterial risk assessment*. In: Proceedings of the
17th Annual Bio-Ontologies Meeting. pp. 18-21, 2014.

C. Goble, A. Gray, L. Harland, K. Karapetyan, A. Loizou, I. Mikhailov, Y. Rankka,
S. Senger, V. Tkachenko, A. Williams, **E. Willighagen**, *Incorporating commercial
and private data into an open linked data platform for drug discovery*. In: The
Semantic Web – ISWC, Lecture Notes in Computer Science. Vol 8219. pp. 65-80.
doi:[10.1007/978-3-642-41338-4_5](https://doi.org/10.1007/978-3-642-41338-4_5), 2013.

C. Brenninkmeijer, C. Evelo, C. Goble, A. Gray, P. Groth, S. Pettifer, R. Stevens,
A. William, **E. Willighagen**, *Scientific Lenses over Linked Data: An Approach to
Support Task Specific Views of the Data*. A Vision. In: Linked Science 2012 - Tackling
Big Data, 2012.

**E.L. Willighagen**, J.E.S. Wikberg, *Linking Open Drug Data to Cheminformatics and
Proteochemometrics*, CEUR Workshop Proceedings In SWAT4LS-2009 - Semantic
Web Applications and Tools for Life Sciences, Vol. 559, 2010.

Zhang, Y., Murray-Rust, P., Dove, M., Glen, R.C., Rzepa, H.S., Townsend, J.A.,
Tyrell, S., Wakelin, J., **Willighagen, E.L.**, 2004. *Jumbo - an XML infrastructure for
eScience*. In: Proceedings of UK e-Science All Hands Meeting 2004. Vol. 1. pp.
930-933, 2004.

Murray-Rust, P., Glen, R.C., Rzepa, H.S., Stewart, J.J.P., Townsend, J.A.,
**Willighagen, E.L.**, Yong, Z., *A semantic grid for molecular science*.
In: Proceedings of the 2003 UK e-Science All Hands Meeting, 2003.

### Book chapters

D.N. Slenter, M. Kutmon, **E.L. Willighagen**, *WikiPathways: Integrating
Pathway Knowledge with Clinical Data*, in: N. Blau, C. Dionisi Vici, C.R.
Ferreira, C. Vianey-Saban, C.D.M. van Karnebeek (Eds.), Physician’s Guide to
the Diagnosis, Treatment, and Follow-Up of Inherited Metabolic Diseases,
Springer International Publishing, UK, Ch. 73, 2022. doi:[10.1007/978-3-030-67727-5_73](https://doi.org/10.1007/978-3-030-67727-5_73)

A.J. Williams, S. Ekins, O. Spjuth, **E. Willighagen**, *Acessing, Using, and
Creating Chemical Property Databases for Computational Toxicology Modeling*,
In: Computational Toxicology (Vol. 929), Springer Science+Business Media, Ch. 10, 2012.

**E. Willighagen**, R. Affentranger, R. Grafström, B. Hardy, N. Jeliazkova,
O. Spjuth, *Interactive Predictive Toxicology with Bioclipse and OpenTox*, In:
Open Source Software in Life Science Research: Practical Solutions to
Common Challenges in the Pharmaceutical Industry and Beyond, Ed. L. 
Harland, M. Forster, Biohealthcare Publishing Ltd, Oxford, UK, Ch. 2, ISBN:
1-907-568972, 2012. doi:[10.1533/9781908818249.35](https://doi.org/10.1533/9781908818249.35)

B. Hardy, A. Affentranger, E. Benfenati, N. Jeliazkova, V. Jeliazkov, G. My-
att, J. Wiseman, G. Stacey, R. Grafström, **E. Willighagen**, C. Helma, *ToxBank:
Supporting Integrated Data Analysis and Servicing of Alternative Testing
Methods in Toxicology*, In: Towards the Replacement of in vivo Repeated Dose
Systemic Toxicity Testing, SEURAT-1, 2011.

R. Guha, O. Spjuth, **E. Willighagen**, *Collaborative Cheminformatics Applications*,
In Collaborative Computational Technologies for Biomedical Research, Ed. S. Ekins,
M.A.Z. Hupcey, and A.J. Williams, John Wiley & Sons,
Inc., Hoboken, USA, ISBN: 9-780-470-63803-3, 2011. doi:[10.1002/9781118026038.ch24](https://doi.org/10.1002/9781118026038.ch24)

**E. Willighagen**, *3D Molecular Representations, In Handbook of Cheminformatics
Algorithms*, Ed. Faulon, J-P. and Bender, A., CRC Press, New York,
US, Ch. 3. ISBN: 1-420-08292-2, 2010. doi:[10.1201/9781420082999-c3](https://doi.org/10.1201/9781420082999-c3)

J-C. Bradley, R. Guha, A. Lang, P. Lindenbaum, C. Neylon, A. Williams,
**E. Willighagen**, *Beautifying Data in the Real World*. In: Beautiful Data - The
Stories Behind Elegant Data Solutions, O’Reilly Media, Inc., Sebastopol,
US, Ch. 16, ISBN: 0-596-15711-8, 2009.

### Editorials

Guha, R., **Willighagen, E.**, Zdrazil, B., & Jeliazkova, N. (2021). What is the role of cheminformatics in a pandemic? *Journal of Cheminformatics*, *13*(1). doi:[10.1186/s13321-021-00491-6](https://doi.org/10.1186/s13321-021-00491-6)

Guha, R., Jeliazkova, N., **Willighagen, E.**, & Zdrazil, B. (2021). Reply to “FAIR chemical structure in the Journal of Cheminformatics.” *Journal of Cheminformatics*, *13*(1). doi:[10.1186/s13321-021-00521-3](https://doi.org/10.1186/s13321-021-00521-3)

Guha, R., & **Willighagen, E.** (2020). Learning cheminformatics. *Journal of Cheminformatics*, *12*(1). doi:[10.1186/s13321-019-0406-z](https://doi.org/10.1186/s13321-019-0406-z)

**Willighagen, E.** (2020). Adoption of the Citation Typing Ontology by the Journal of Cheminformatics. *Journal of Cheminformatics*, *12*(1). doi:[10.1186/s13321-020-00448-1](https://doi.org/10.1186/s13321-020-00448-1)

**Willighagen, E.**, Jeliazkova, N., & Guha, R. (2019). Journal of Cheminformatics, ORCID, and GitHub. *Journal of Cheminformatics*, *11*(1). doi:[10.1186/s13321-019-0365-4](https://doi.org/10.1186/s13321-019-0365-4)

Guha, R., & **Willighagen, E.** (2017). Helping to improve the practice of cheminformatics. *Journal of Cheminformatics*, *9*(1). doi:[10.1186/s13321-017-0217-z](https://doi.org/10.1186/s13321-017-0217-z)

**Willighagen, E. L.**, & Brändle, M. P. (2011). Resource description framework technologies in chemistry. *Journal of Cheminformatics*, *3*(1). doi:[10.1186/1758-2946-3-15](https://doi.org/10.1186/1758-2946-3-15)


## Teaching experience

### C-roles

`2013-present`
Course coordinator, **PRA3006: Programming in the Life Sciences**. Maastricht University

`2018-2020`
Block coordinator, **MBS1015: Scientific Programming**. Maastricht University

`2017-2018`
Block coordinator, **BMS1004: Biomedical Approaches**. Development. Maastricht University

`2012-2018`
Track coordinator, **BMW2007: Bioinformatics and Systems Biology**. Maastricht University

### B-roles

`2016-2022`
Block planning group member, **BBS1001: Building Blocks of Life**. Maastricht University

`2015-2016`
Block planning group member, **BMW1006: Experts Collaborate**. Maastricht University

`2007`
Teacher, **Homology Modelling**. CUBIC, Cologne University

### A-roles

`2016-present`
**Mentor Year 1-3, Biomedical Sciences**. Maastricht University

`2015-2016`
**Mentor Year 1, Biomedical Sciences**. Maastricht University

`2014-2019`
Problem-based Learning (PBL) **Tutor, Biomedical Sciences**. Maastricht University

## Thesis promotorship and supervision

### Ph.D. co-promotor

`2017-2024`
**Denise Slenter**, Maastricht University. <i class="fa-solid fa-building-columns"></i> [10.26481/dis.20240624ds](https://doi.org/10.26481/dis.20240624ds)

`2017-2024`
**Marvin Martens**, Maastricht University. <i class="fa-solid fa-building-columns"></i> [10.26481/dis.20240129mm](https://doi.org/10.26481/dis.20240129mm)

`2019-2024`
**Andra Waagmeester**, Maastricht University. <i class="fa-solid fa-building-columns"></i> [10.26481/dis.20240116aw](https://doi.org/10.26481/dis.20240116aw)

`2018-2023`
**Ryan Miller**, Maastricht University. <i class="fa-solid fa-building-columns"></i> [10.26481/dis.20221206rm](https://doi.org/10.26481/dis.20221206rm)

### Ph.D. supervision

`2024-present`
**Sidra Adil**, Maastricht University.

`2022-present`
**Jente Houweling**, Maastricht University.

`2022-present`
**Javier Millán Acosta**, Maastricht University.

`2020-2025`
**Ammar Ammar**, Maastricht University.

## Awards and honors

`2021`
**Runner Up Prize, Open Initiatives Trophies**
From the jury-report: *"Egon is someone with high standards who really tries to
hold himself accountable to these as well. He is not afraid to point his finger at
the elephant in the room. We need people like Egon as a long time driving force
for cultural change."*

`2013`
**Royal Society of Chemistry eScience Hero Award**
For the contributions open source cheminformatics, in particular the Chemistry
Development Kit, $2500. [<i class="fa-solid fa-newspaper"></i> press release](https://www.rsc.org/news-events/articles/2013/04-april/rsc-escience-heroes-rewarded-through-microsoft-prize/)

`2007`
**Blue Obelisk Award**
For achievements in promoting Open Data, Open Source, and Open Standards
in cheminformatics.

## Grants and funding

<i class="fa-regular fa-money-bill-1"></i> = grant, UM = Maastricht University

### Research grants

`2024-present`
**INTOXICOM Implementation Study**. <!-- Coordinator: UM. Other: . -->
ELIXIR Europe. Sum: 59 k€, Person: ?? k€. *Co-PI with Marvin Martens*

`2023-present`
**TGX-MAP: Transcriptomics in Risk Assessment**. <!-- Coordinator: Leiden University. Other: UM, a few more. -->
EFSA. Sum: ?? k€, UM: 276 k€. *Co-PI with Tooba Abbassi-Daloii and Marvin Martens*.

`2012-present`
**The Virtual Human Platform for Safety Assessment**. 
Dutch Research Council (NWO), [<i class="fa-regular fa-money-bill-1"></i> NWA 1292.19.272](https://www.nwo.nl/projecten/nwa129219272). Sum: 10.4 M€, UM: ?? k€. *Co-PI with Chris Evelo*.

`2024-2025`
**Chemistry Development Kit 2024**. <!-- Coordinator: Egon Willighagen. Other: Hogeschool Utrecht. --> Dutch Research Council (NWO). Sum: 50 k€, Person: 25 k€. *Sole PI*. doi:[10.3897/rio.10.e124884](https://doi.org/10.3897/rio.10.e124884)

`2022-2023`
**Lipid Pathway Curation**. <!-- Coordinator: Cardiff University. Other: UM -->
ELIXIR Europe. Sum: ?? k€, UM: 5.7 k€. *Co-PI with Denise Slenter*.

`2022-2023`
**BridgeDb and Wikidata: a powerful combination generating interoperable open research**, <!-- Coordinator: UM. -->
Dutch Research Council (NWO), [<i class="fa-regular fa-money-bill-1"></i> 203.001.121](https://www.nwo.nl/en/projects/203001121). Sum: 50 k€, UM: 50 k€. *Sole PI*. doi:[10.3897/rio.8.e83031](https://doi.org/10.3897/rio.8.e83031)

`2020-2024`
**Computing infrastructure for the definition, performance testing and implementation of safe-by-design approaches in nanotechnology supply chains (SbD4nano)**, <!-- Coordinator: ITENE. Other: UM -->
European Commission H2020. [<i class="fa-regular fa-money-bill-1"></i> 862195](https://cordis.europa.eu/project/id/862195). Sum: 5.98 M€, UM: 343 k€. *Sole PI*.

`2019-2021`
**Scholia**, <!-- Coordinator: University of Virginia. Other: UM -->
Alfred P. Sloan Foundation. [<i class="fa-regular fa-money-bill-1"></i> G-2019-11458](https://sloan.org/grant-detail/g-2019-11458). Sum: 496 k€, UM: 75 k€. *Sole PI*. doi:[10.3897/rio.5.e35820]https://doi.org/10.3897/rio.5.e35820)

`2019-2023`
**Risk Governance of Nanotechnology (RiskGONE)**, <!-- Coordinator: NILU. Other: UM -->
European Commission H2020. [<i class="fa-regular fa-money-bill-1"></i> 814425](https://cordis.europa.eu/project/id/814425). Sum: 5.00 M€, UM: 221 k€. *Sole PI*.

`2019-2023`
**Innovative Nanoinformatics models and tools: towards a Solid, verified and Integrated Approach to Predictive (eco)Toxicology (NanoSolveIT)**, <!-- Coordinator: NovaMechanics. Other: UM -->
European Commission H2020. [<i class="fa-regular fa-money-bill-1"></i> 814572](https://cordis.europa.eu/project/id/814572). Sum: 7.49 M€, UM: 407 k€. *Sole PI*.

`2018-2022`
**The European Nanotechnology Community Informatics Platform: Bridging data and disciplinary gaps for industry and regulators (NanoCommons)**, <!-- Coordinator: UoB. Other: UM -->
European Commission H2020. [<i class="fa-regular fa-money-bill-1"></i> 731032](https://cordis.europa.eu/project/id/731032). Sum: 5.59 M€, UM: 291 k€. *Sole PI*.

`2014-2017`
**eNanoMapper - A Database and Ontology Framework for Nanomaterials Design and Safety Assessment**, <!-- Coordinator: EC. Other: UM -->
European Commission FP7. [<i class="fa-regular fa-money-bill-1"></i> 604134](https://cordis.europa.eu/project/id/604134). Sum: 4.98 M€, UM: 843 k€. *Sole PI*.

<!--
\CVItem{2015, \textit{AMBIT}, Cefic Long-Range Research Intitiative}{
Collaborator, work on Chemistry Development Kit, EEM9.3-IC, 16 k€.
}

\CVItem{2020, \textit{EpiLipidNET}, COST Action}{
co-PI.
}
\CVItem{2011, \textit{General research support}, KI Fonden}{
PI, personal grant, $\sim$2900 euro (24 kSEK).
}

\CVItem{2000 - 2019, Various small project and travel grants}{
Four travel grants (Specs\&BioSpecs, eCheminfo Bursary Award,
and twice Wallenberg Fonden), total $\sim$5025 euro.
Four times Google Summer of Code (3x \$ 5000, 1x \$ 6600) and twice a Dutch "Programmeerzomer"
grant (4 kEuro each). Various projects with ELIXIR (Metabolomics Implementation Study,
Bio.tools project; $\sim$5 kEuro).
}
-->

## Talks

### Keynotes

`2022`
**OpenTox Virtual Conference 2022**. Online.

`2015`
**International Conference on Biomedical Ontology**. Lisbon, Portugal

### Invited talks

`2023`
**University of Groningen**. *Open Science Community Groningen Summer Event*. Groningen, The Netherlands

**OECD**. *OECD "FAIR research output for nanosafety" meeting*. Paris, France

`2019`
**Beilstein Open Science 2019**. Rüdesheim, Germany

`2017`
**AgreenSkills+ Annual Meeting**. Paris, France

`2015`
**System Biology in Nanosafety Research**. Stockholm, Sweden

`2014`
**Royal Society of Chemistry**. *What’s in a Name?*. London, UK

`2009`
**GDCh-Wissenschafts-forum Chemie 2009**. Frankfurt, Germany

`2007`
**GDCh-Wissenschaftsforum Chemie 2007**. Ulm, Germany

## Memberships

`1993-???,???-present`
**KNCV**

`2023-2024`
**American Chemical Society**

## Professional service

### Scientific committees

`2024-present`
Titular Member, International Union of Pure and Applied Chemistry, CPCDS

`2023-present`
Advisor Board Member, The Netherlands Reproducibility Network

`2022-present`
Advisory Board Member, NFDI4Chem

`2022-present`
Founder of the Open Science Community Maastricht

`2014-present`
Board Member, Stichting Chemische Congressen 6

`2020-2022`
Scientific Advisor, International Advisory Board of the NanoFabNet Project

`2018-2024`
Working Group Chair, Database Working Group, EU NanoSafety Cluster

`2013-2026`
Working Group Chair, Database Working Group, EU NanoSafety Cluster

### Editorial Boards

`2025-present`
Guest-editor, Journal of Cheminformatics (Springer Nature. Top10 journal)

`2016-2021`
Editor-in-Chief, Journal of Cheminformatics (Springer Nature. Top10 journal)

`2010`
Guest-editor, Journal of Cheminformatics (Springer Nature. Top10 journal)

`2009-2016`
Invited Expert, World Wide Web Consortium (W3C)

`2009-2011`
CINF Programming Committee Member, American Chemical Society

`2007-2011`
Advisor, ChemSpider.com Advisory Group

## Media

### Opinions

`2020`
Caspar van Lissa c.s. **Overheid wees transparant in strijd tegen het coronavirus**. NRC (2020) [https://www.nrc.nl/nieuws/2020/04/27/overheid-wees-transparant-in-strijd-tegen-het-coronavirus-a3997982](https://www.nrc.nl/nieuws/2020/04/27/overheid-wees-transparant-in-strijd-tegen-het-coronavirus-a3997982)

`2019`
Egon Willighagen. **Universiteiten mogen hun rol in de samenleving niet uitbesteden**. ScienceGuide (2019) [https://www.scienceguide.nl/2019/11/universiteiten-mogen-hun-rol-in-de-samenleving-niet-uitbesteden/](https://www.scienceguide.nl/2019/11/universiteiten-mogen-hun-rol-in-de-samenleving-niet-uitbesteden/)

### Interviews

`2019`
Laura Howes. **Chemistry data should be FAIR, proponents say. But getting there will be a long road**, C&EN (2019) [https://cen.acs.org/policy/publishing/Chemistry-data-should-FAIR-proponents/97/i35](https://cen.acs.org/policy/publishing/Chemistry-data-should-FAIR-proponents/97/i35)

Sicco de Knecht. **So what about editor compensation?**, ScienceGuide (2019) [https://www.scienceguide.nl/2019/04/so-what-about-editor-compensation/](https://www.scienceguide.nl/2019/04/so-what-about-editor-compensation/)

`2018`
**“Als je geen open access wilt, zeg dat dan gewoon hardop!”** ScienceGuide (2018) [https://www.scienceguide.nl/2018/12/als-je-geen-open-access-wilt-zeg-dat-dan-gewoon-hardop/](https://www.scienceguide.nl/2018/12/als-je-geen-open-access-wilt-zeg-dat-dan-gewoon-hardop/)

Arjen Dijkgraag. **Open accessstress**. C2W (2018), [https://www.sciencelink.net/nieuws-and-verdieping/open-accessstress/8374.article](https://www.sciencelink.net/nieuws-and-verdieping/open-accessstress/8374.article)

`2017`
Puck Moll. **Focus op software**, C2W (2017), [https://www.sciencelink.net/nieuws-and-verdieping/focus-op-software/5919.article](https://www.sciencelink.net/nieuws-and-verdieping/focus-op-software/5919.article)

### News

`2021`
Maurice Timmermans, **Twee UM’ers ontvangen NWO-beurzen voor open science**, Observant (2021) [https://www.observantonline.nl/Home/Artikelen/id/57779/twee-um-ers-ontvangen-nwo-beurzen-voor%C2%A0open-science](https://www.observantonline.nl/Home/Artikelen/id/57779/twee-um-ers-ontvangen-nwo-beurzen-voor%C2%A0open-science)

`2020`
Cleo Freriks, **UM researchers: corona is given high priority**, Observant (2020) [https://www.observantonline.nl/english/Home/Articles/id/43066](https://www.observantonline.nl/english/Home/Articles/id/43066)

`2019`
Andrea Widener. **Societies and scientists react to open access Plan S**, C&EN (2019) [https://cen.acs.org/policy/publishing/Societies-scientists-react-open-access/97/web/2019/02](https://cen.acs.org/policy/publishing/Societies-scientists-react-open-access/97/web/2019/02)

Sicco de Knecht. **Germans to close “transformative” deal with SpringerNature**, ScienceGuide (2019) [https://www.scienceguide.nl/2019/08/germans-to-close-transformative-deal-with-springernature/](https://www.scienceguide.nl/2019/08/germans-to-close-transformative-deal-with-springernature/)

`2018`
Puck Moll. **Nieuwe generatie**, C2W (2017), [https://www.sciencelink.net/nieuws-and-verdieping/nieuwe-generatie/6551.article](https://www.sciencelink.net/nieuws-and-verdieping/nieuwe-generatie/6551.article)

`2014`
Wendy Degens, Cleo Freriks, Maurice Timmermans, **“I see bright researchers leaving the university”**, Observant (2014) [https://www.observantonline.nl/english/Home-old/Articles/articleType/ArticleView/articleId/2111/I-see-bright-researchers-leaving-the-university](https://www.observantonline.nl/english/Home-old/Articles/articleType/ArticleView/articleId/2111/I-see-bright-researchers-leaving-the-university)

### International Newsletters

`2022`
**ELIXIR launches new Toxicology Community**, ELIXIR Europe News (2022) [https://elixir-europe.org/news/new-toxicology-community](https://elixir-europe.org/news/new-toxicology-community)

`2021`
**Niets afschaffen, wel verbreden**, NWO Magazine (2021) [https://www.nwo.nl/niets-afschaffen-wel-verbreden](https://www.nwo.nl/niets-afschaffen-wel-verbreden)

## Languages

Native: Dutch

Advanced: English

Basic: German

<br/>Last updated: April 2025<br/><br/>
